client/src/main/java/io/getstream/chat/android/client/helpers/AttachmentHelper.kt

```diff
@@ -0,0 +1,31 @@
+package io.getstream.chat.android.client.helpers
+
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.client.utils.SystemTimeProvider
+import org.apache.commons.validator.routines.UrlValidator
+
+class AttachmentHelper(private val systemTimeProvider: SystemTimeProvider = SystemTimeProvider()) {
```

Comment:
Can this class be internal? We already have its functionality exposed through `ChatClient`.

Reply:
I think it can expose this class and remove `ChatClient::hasValidUrl()` method.
We don't need it there